.github/workflows/ci-build-release-napi.yml

```diff
@@ -38,16 +38,24 @@ jobs:
           path: pkg/mac/build
           key: ${{ runner.os }}-${{ matrix.arch }}-mac-${{ hashFiles('pkg/mac/build-cpp-deps-lib.sh') }}
 
+      - name: Add arch env vars
+        run: |
+          if [ "${{ matrix.arch }}" = "x64" ]; then
+            echo "ARCH=x86_64" >> $GITHUB_ENV
+          else
+            echo "ARCH=${{ matrix.target }}" >> $GITHUB_ENV
```

Comment:
What is `matrix.target` referred to here?

Reply:
Oh, Wrong written here. 

As for why CI can pass, it is because the default value of arch in the `common.sh` is arm64